cache/clock_cache.h

```diff
@@ -305,11 +305,11 @@ struct ClockHandle : public ClockHandleBasicData {
   // state of the handle. The meta word looks like this:
   // low bits                                                     high bits
   // -----------------------------------------------------------------------
-  // | acquire counter          | release counter           | state marker |
+  // | acquire counter      | release counter     | hit bit | state marker |
   // -----------------------------------------------------------------------
 
   // For reading or updating counters in meta word.
-  static constexpr uint8_t kCounterNumBits = 30;
+  static constexpr uint8_t kCounterNumBits = 29;
```

Comment:
I believe this doesn't need to change, because there are only 3 state bits. 30 * 2 + 1 + 3 == 64